### ISSUE TITLE
Feature request: Support for ASL

### ISSUE THREAD

**Issue by ChristianNSchmitz** [nipy/heudiconv#504]
Dear nipy-team,

We would like to ask you if you are planning support of heudiconv for ASL (newly published on BIDS v1.5.0 on 2021-02-23)?

Best wishes and thanks!
Christian

**Comment by yarikoptic** [comment]
yeah, would be nice... never dealt with that beast yet so have no clue on its "specifics" -- any sample dicoms available anywhere. I wonder what special treatment would it need if any? i.e. I see that some specific heuristics (like reproin) might need to be adjusted but heudiconv code itself might not need?

**Comment by neurolabusc** [comment]
My response is similar to [issue 514](https://github.com/nipy/heudiconv/issues/514), except [BEP005](https://docs.google.com/document/d/15tnn5F10KpgHypaQJNNGiNKsni9035GtDqJzWqkkP6c) instead of BEP009. The files you link to only show the BIDS/NIfTI solution, without the DICOM images that they should be derived from. Without exemplars of input and output, it is hard to support this. 

I did generate [my own ASL data](https://osf.io/td4bx/) and attempt to comprehensively extract all the [ASL information I can](https://github.com/rordenlab/dcm2niix/tree/development/BIDS) from these datasets. However, I only have Siemens equipment, and the data stored by Siemens is very different from what BEP005 expects. I think the Philips and [GE](https://groups.google.com/g/comp.protocols.dicom/c/8MWkndr_FVc?pli=1) ASL-specific DICOM tags are even more limited. It seems that BEP005 was developed based on what information is required to process this modality, not the information provided in the DICOMs for this modality. Therefore, users will need to complete the JSON files themselves.

**Comment by yarikoptic** [comment]
@HenkMutsaerts mentioned that some metadata might be available in Phoenix reports for Siemens (IIRC support was added in #489 and may be elsewhere), and in XX files for Philips.
